include/experimental/__p1673_bits/blas1_linalg_add.hpp

```diff
@@ -121,15 +151,14 @@ void add(
   std::experimental::mdspan<ElementType_y, std::experimental::extents<ext_y ...>, Layout_y, Accessor_y> y,
   std::experimental::mdspan<ElementType_z, std::experimental::extents<ext_z ...>, Layout_z, Accessor_z> z)
 {
+  static_assert(z.rank() <= 2);
```

Comment:
This is harmless and please don't trouble yourself, but generally I think we want to specify the rank bound as a Constraint ("does not participate in overload resolution"), rather than a Mandate (`static_assert`).

Reply:
Actually... don't worry about this for now, please : - ) .  I'll file an issue with the proposal itself, since it's not 100% clear to me whether it specifies this as a Constraint or Mandate.